frontend/src/main/java/starter/project/frontend/CarController.java

```diff
@@ -8,25 +8,19 @@
 
 @RestController
 public class CarController {
-
-    private List<Car> currentCarList = new ArrayList<>();
     Map<Integer, Car> map = new HashMap<>();
 
     @PostMapping(path = "/car", consumes = {"application/json"})
-    public List<Car> addCar(@RequestBody Car car) {
-        currentCarList.add(car);
-        return currentCarList;
+    public Map<Integer,Car> addCar(@RequestBody Car car) {
+        map.put(car.getId(),car);
+        return map;
     }
+
     @GetMapping(path = "/car/{carId}")
     public Car getCar(@PathVariable("carId") int carId) {
-        for (Car car : currentCarList) {
-            if (car.getId() == (carId)) {
+     Car car =  map.get(carId);
```

Comment:
We could simply do `return map.get(carId)` as we don't really do anything to the object we get back

Reply:
This was marked resolved but not changed? Is there justification to keeping the variable